CoherentTasks/CoherentTasks/ISBN.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoherentTasks
+{
+    class ISBN
+    {
+        public static void Execute()
+        {
+            Console.Write("Please enter the first 9 digits of ISBN-10 idientifier: ");
+            string isbn = Console.ReadLine();
+
```

Comment:
You have to output the whole ISBN, not only the check digit, there is possible that check value will be 10, need to change it to 'X' symbol